### PR TITLE
Added article for v9 to v10 upgrade on Cloud

### PR DESCRIPTION
**Pull request by eshanrnh**
I have named the article title **Upgrading an Umbraco 9 Cloud project to Umbraco 10** rather than Migrating cz we are upgrading from old version to new version.

### CODE REVIEW

**Review by sofietoft — COMMENTED** [review]
Some initial notes: (just adding them now, as I might not finish the guide before heading home for today)

- No need to use the version number whenever referencing the project - "Umbraco 9". The 8 to 9 article used an approach with having 2 Cloud project. That's not the case this time, so I don't think it's necessary to reference Umbraco 9 anymore. Perhaps "just" call it "the project" instead.

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+* A backup of your Umbraco 9 project database.
+
+:::note
+We strongly recommend having at least 2 environments on your project. If something fails during the upgrade, the Development environment can be removed and added again to start over with the upgrade process.
+:::
+
+## Step 1: Content Migration
+
+* Create a backup of the database from your project, see the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article or clone down the project and take a backup of the local Database (Ensure to restore the content from your cloud environment).
```
Comment (jonat123) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:23]:
```suggestion
* Create a backup of the database from your project, see the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article or clone down the project and take a backup of the local Database (Make sure to restore the content from your cloud environment).
```
```suggestion
* Create a backup of the database from your project, see the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article or clone down the project and take a backup of the local Database (Ensure to restore the content from your cloud environment).
```

Comment (jonat123) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:23]:
```suggestion
* Create a backup of the database from your project, see the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article or clone down the project and take a backup of the local Database (Make sure to restore the content from your cloud environment).
```
```suggestion
* Create a backup of the database from your project, see the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article or clone down the project and take a backup of the local Database (Ensure to restore the content from your cloud environment).
```

**Comment by jonat123** [comment]
@eshanrnh Looks good to me :) 

I made one suggestion (Not sure why it decided to make two...) :)

**Comment by jonat123** [comment]
@eshanrnh Looks good to me :) 

I made one suggestion (Not sure why it decided to make two...) :)

**Comment by eshanrnh** [comment]
> @eshanrnh Looks good to me :)
> 
> I made one suggestion (Not sure why it decided to make two...) :)

Github does the same with me too - 2 suggestions/comments (a little annoying at times) 😀

Incorporated the suggestion.

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+## Step 1: Content Migration
+
+* Create a backup of the database from your project, see the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article or clone down the project and take a backup of the local Database (Make sure to restore the content from your cloud environment).
+
+* On the Cloud portal, go to the project that you wish to upgrade and navigate to **Settings** -> **Advanced**. Scroll down to **Runtime Settings** section and **Enable .NET 6 for your Umbraco 9 install** for each environment of your cloud project.
+
+    ![Runtime Settings](images/Runtime-Settings.png)
+
+* Clone down the **Development** environment from the Umbraco Cloud project, build and run the project locally and make sure to log in to the backoffice.
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:29]: Should there also be a step for restoring content? I guess you would wanna test the upgrade against that, right?

```suggestion
* Clone down the **Development** environment from the Umbraco Cloud project.
* Build and run the project locally.
* Make sure to log in to the backoffice.
* Restore content from your Cloud enviroment.
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+* An Umbraco 9 Cloud project running **the latest version of Umbraco 9** with **at least 2 environments**.
+
+* A backup of your Umbraco 9 project database.
+
+:::note
+We strongly recommend having at least 2 environments on your project. If something fails during the upgrade, the Development environment can be removed and added again to start over with the upgrade process.
+:::
+
+## Step 1: Content Migration
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:21]:
```suggestion
## Step 1: Clone down your environment
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+* Create a backup of the database from your project, see the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article or clone down the project and take a backup of the local Database (Make sure to restore the content from your cloud environment).
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:23]:
```suggestion
* Create a backup of the database
  * Directly from your environment. See the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article 
  * Or clone down and restore the project, and take a backup of the local Database.
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+* On the Cloud portal, go to the project that you wish to upgrade and navigate to **Settings** -> **Advanced**. Scroll down to **Runtime Settings** section and **Enable .NET 6 for your Umbraco 9 install** for each environment of your cloud project.
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:25]:
```suggestion
* Go to the project in the Cloud portal.
* Navigate to **Settings** -> **Advanced**
* Scroll down to the **Runtime Settings** section.
* **Enable .NET 6 for your Umbraco 9 install** for each environment of your Cloud project.
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+    ![Runtime Settings](images/Runtime-Settings.png)
+
+* Clone down the **Development** environment from the Umbraco Cloud project, build and run the project locally and make sure to log in to the backoffice.
+
+## Step 2: Upgrade the project locally using Visual Studio
+
+* Open your project in Visual Studio.
+
+* Right-click your project solution in the **Solution Explorer** and select **Properties**.
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:35]:
```suggestion
* Right-click your project solution in the **Solution Explorer**.
* Select **Properties**.
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+    ![Solution Explorer](images/Solution-Explorer.png)
+
+* In the **General** section of the **Application** tab, select **.Net 6.0** from the **Target Framework** drop-down.
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:38]:
```suggestion
* Select **.Net 6.0** from the **Target Framework** drop-down in the **General** section of the **Application** tab.
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+    ![Target Framework](images/Target-Framework.png)
+
+* Go to **Tools** > **NuGet Package Manager** > **Manage NuGet Packages for Solution...**.
+
+* In the **NuGet Package manager Solution**, go to the **Installed** tab and choose **Umbraco.Cms**.
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:43]:
```suggestion
* Go to the **Installed** tab In the **NuGet Package manager Solution**.
* Choose **Umbraco.Cms**
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+
+* Select your project, choose **10.0.0** from the **Version** drop-down and click **Install** to upgrade your project to version 10.
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:45]:
```suggestion
* Select your project.
* Choose **10.0.0** from the **Version** drop-down.
Click **Install** to upgrade your project to version 10.
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+    ![Nuget Version Install](images/Nuget-Version-Install.png)
+
+* Similarly, update the below packages referenced in your project to the latest versions available on NuGet either through the **NuGet Package Manager** or **Package Manager Console (PowerShell)**:
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:48]:
```suggestion
* Update the below packages referenced in your project to the latest versions available on NuGet either through the **NuGet Package Manager** or **Package Manager Console (PowerShell)**:
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+                    .ConfigureUmbracoDefaults()
+                    .ConfigureWebHostDefaults(webBuilder =>
+                    {
+                        webBuilder.UseStaticWebAssets();
+                        webBuilder.UseStartup<Startup>();
+                    });
+        }
+    ```
+
+* To re-enable the appsettings IntelliSense, update your schema reference in the **appsettings.json**, **appsettings.Development.json**, **appsettings.Production.json**, and **appsettings.Staging.json** files from:
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:81]:
```suggestion
* Re-enable the appsettings IntelliSense by updating your schema reference in the **appsettings.json**, **appsettings.Development.json**, **appsettings.Production.json**, and **appsettings.Staging.json** files from:
```

File: Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+    "$schema": "./umbraco/config/appsettings-schema.json",
+    ```
+
+    To:
+
+    ```json
+    "$schema": "./appsettings-schema.json",
+    ```
+
+* Remove the following files and folders *manually* from your local project and on the cloud **Development** environment through [KUDU](../../Set-Up/Power-Tools/index.md) both from the `repository` and `wwwroot` folders:
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md:93]: This is a long item.
Would it make sense to split it in two?
One for local, and one for KUDU?

**Review by sofietoft — CHANGES_REQUESTED** [review]
I'm going to be running through the guide now.

I've made quite a few suggestion here - mainly it's cleaning up the list, so it always start with an action word, and only has a single action per item.
Let's try to make that our default 💪

**Pull request merged**